### PR TITLE
RHCLOUD-29867 Text changes on Notifications overview page

### PR DESCRIPTION
**Pull request by aferd**

### CODE REVIEW

**Comment by aferd** [comment]
Jira [RHCLOUD-29867](https://issues.redhat.com/browse/RHCLOUD-29867)

File: src/pages/Notifications/Overview/CustomDataListItem.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ interface CustomDataListItemProps {
   heading: string;
   linkTitle?: string;
   linkTarget?: string;
-  expandableContent: string;
+  expandableContent: ReactNode;
```
Comment (aneelac22) [inline src/pages/Notifications/Overview/CustomDataListItem.tsx]: This should be any, I see it is string in some places

**Review by aneelac22 — CHANGES_REQUESTED** [review]

File: src/pages/Notifications/Overview/CustomDataListItem.tsx (hunk 1 above)
Comment (aferd) [reply src/pages/Notifications/Overview/CustomDataListItem.tsx]: Typescript here won't allow `any`, it gets very angry. It was yelling at me about it being an Element where I added the `<span>` to be able to add the link in the `expandableContent`.

File: src/pages/Notifications/Overview/Page.tsx [hunk 2]
```diff
@@ -288,9 +289,23 @@ export const NotificationsOverviewPage: React.FunctionComponent = () => {
                         isBeta() ? '/preview' : ''
                       }/${getBundle()}/notifications/integrations`
                 }
-                expandableContent="Notifications and integrations services work together to transmit messages to third-party application
-                    endpoints, such as instant messaging platforms and external ticketing systems, when triggering events occur. Integrations
-                    include Splunk, Slack, ServiceNow, and more."
+                expandableContent={
+                  <span>
```
Comment (aferd) [inline src/pages/Notifications/Overview/Page.tsx:293]: here.

File: src/pages/Notifications/Overview/CustomDataListItem.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ import {
   UserIcon,
   UsersIcon,
 } from '@patternfly/react-icons';
-import React from 'react';
+import React, { ReactNode } from 'react';
```
Comment (aneelac22) [inline src/pages/Notifications/Overview/CustomDataListItem.tsx]:
```suggestion
import React from 'react';
```

File: src/pages/Notifications/Overview/CustomDataListItem.tsx (hunk 1 above)
Comment (aneelac22) [inline src/pages/Notifications/Overview/CustomDataListItem.tsx]:
```suggestion
  expandableContent: React.ReactNode;
```

File: src/pages/Notifications/Overview/Page.tsx [hunk 4]
```diff
@@ -261,9 +263,8 @@ export const NotificationsOverviewPage: React.FunctionComponent = () => {
                 linkTitle="Go to Notification Preferences"
                 linkTarget={`${
                   isBeta() ? '/preview' : ''
-                }/${getBundle()}/notifications/user-preferences`}
-                expandableContent="This service allows you to opt-in and out of receiving notifications. Your Organization
-                            Administrator has configured which notifications you can or can not receive in their Settings."
+                }/${getBundle()}/notifications/my-notification-preferences`}
```
Comment (InsaneZein) [inline src/pages/Notifications/Overview/Page.tsx]: the url should still be `/notifications/user-preferences`

Comment (aferd) [reply src/pages/Notifications/Overview/Page.tsx]: good catch!

**Review by aneelac22 — APPROVED** [review]

**Pull request merged**